src/Models/LoginModel.php

```diff
@@ -56,6 +56,12 @@ public function recordLoginAttempt(
         ?string $userAgent = null,
         $userId = null
     ): void {
+        $this->disableDBDebug();
```

Comment:
Can you explain the need for this?

Reply:
`ErrorException` was thrown on Oracle database because of the SQL error.